Resources/Locale/en-US/game-ticking/game-presets/preset-zombies.ftl

```diff
@@ -15,6 +15,7 @@ zombie-infection-underway = Your blood begins to thicken
 zombie-alone = You feel entirely alone.
 
 zombie-shuttle-call = We have detected that the undead have overtaken the station. Dispatching an emergency shuttle to collect remaining personnel.
+zombie-shuttle-unavailable = Shuttle unavailable due to high undead percentage.
```

Comment:
Isn't this message for when someone tries to recall?

```suggestion
zombie-shuttle-unavailable = Shuttle recall is unavailable due to high undead percentage.
```

Reply:
i suppose it might be too long but we'll see